proto/message_api/v1/authn.proto

```diff
@@ -13,30 +13,31 @@ option go_package = "./message_api/v1";
 // Signature represents a generalized public key signature,
 // defined as a union to support cryptographic algorithm agility.
 message Signature {
+    oneof union {
+        ECDSACompact ecdsa_compact = 1;
+    }
+
     // ECDSA signature in its compact format
     message ECDSACompact {
         bytes bytes = 1; // compact representation [ R || S ], 64 bytes
         uint32 recovery = 2; // recovery bit
     }
-    oneof union {
-        ECDSACompact ecdsa_compact = 1;
-    }
-}
-
-// Secp256k1 public key in uncompressed point format.
-message Secp256k1Uncompresed {
-    // uncompressed point with prefix (0x04) [ P || X || Y ], 65 bytes
-    bytes bytes = 1; 
 }
 
 // PublicKey represents a generalized public key,
 // defined as a union to support cryptographic algorithm agility.
 message PublicKey {
-    uint64 timestamp = 1;
+    uint64 created_ns = 1;
```

Comment:
Given that we are converting back and forth between the existing crypto protos and these, would be nice to have them be as similar as possible.

Reply:
OK, I can roll this back.

Reply:
Right now the only change is the optional signature, which doesn't prevent the translation between our native crypto types and these new protos directly.

Reply:
I wasn't clear which proto types will be used for what. But I guess with TS ducktyping they are interchangeable as long as they are structurally the same.